wpeverest/everest-forms
# Feature - Beaver Builder Compatibility

### All Submissions:






### Changes proposed in this Pull Request:

Previously, there was no direct compatibility of **Everest Forms** with **Beaver Builder**. This PR adds the **Everest Forms** Widget for in **Beaver Builder** to directly integrate it.

Closes # .

### How to test the changes in this Pull Request:

1. Create a **Page** using **Beaver Builder**
2. Drag and drop the **Everest Forms** Widget from the **Widgets list**
3. And select the desired form.
4. Check whether the form is working as expected or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Beaver Builder Compatibility.
